resources/airflow.rb

```diff
@@ -1045,7 +1045,7 @@ def self.create_sens_lat_load_actuator_and_equipment(model, name, space, is_late
   def self.create_ducts_objects(model, runner, building, ducts, mech_vent, tin_sensor, pbar_sensor, adiabatic_const, air_loop, duct_programs, duct_lks, air_loop_objects)
     return true if ducts.size == 0 # No ducts
 
-    duct_zones = ducts.map { |duct| duct.zone }.uniq!
+    duct_zones = ducts.map { |duct| duct.zone }.uniq
```

Comment:
No idea how this code worked before...

Reply:
Yeah.. I googled for nuances between w/ bang `!` and w/o `!`. With `!`,  it only worked when it did remove repeating stuff (otherwise it will return `nil`), maybe it worked because supply and return ducts are always placed in the same zone, so there's always a repeating element to be removed. Anyways, we are no longer in dangerous status!